core/src/main/com/github/mmorihiro/core/ArrayActor.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.mmorihiro.core
+
+import com.badlogic.gdx.graphics.g2d.Batch
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer
+import com.badlogic.gdx.scenes.scene2d.Actor
+
+
+class ArrayActor : Actor() {
+    private val arraySize = 20
+    private var array = Array(arraySize) { BooleanArray(arraySize) { false } }
+    private val renderer = ShapeRenderer()
+    private val size = 10f
+
+    override fun draw(batch: Batch?, parentAlpha: Float) {
+        batch!!.end()
```

Comment:
If I don't end `batch`, rendering of buttons will be strange because `ShapeRenderer` also has its own batch
 and using two batches is wrong in libgdx.